--- a/package/myapps/passwall-dependencies/v2ray-geodata/Makefile
+++ b/package/myapps/passwall-dependencies/v2ray-geodata/Makefile
@@ -12,22 +12,22 @@
 
 include $(INCLUDE_DIR)/package.mk
 
-GEOIP_VER:=202204210050
+GEOIP_VER:=202204252212
 GEOIP_FILE:=geoip.dat.$(GEOIP_VER)
 define Download/geoip
-  URL:=https://github.com/v2fly/geoip/releases/download/$(GEOIP_VER)/
+  URL:=https://github.com/Loyalsoldier/v2ray-rules-dat/releases/download/$(GEOIP_VER)/
   URL_FILE:=geoip.dat
   FILE:=$(GEOIP_FILE)
-  HASH:=9414426b105894bbacd6e913d180e4d86c32bd08d468f08b8ea8d4f9ecd6674c
+  HASH:=3659d8c74f02a88567b8843710d72758de4ed4e0ec09bd42a05029e02744a04d
 endef
 
-GEOSITE_VER:=20220425025949
-GEOSITE_FILE:=dlc.dat.$(GEOSITE_VER)
+GEOSITE_VER:=202204252212
+GEOSITE_FILE:=geosite.dat.$(GEOSITE_VER)
 define Download/geosite
-  URL:=https://github.com/v2fly/domain-list-community/releases/download/$(GEOSITE_VER)/
-  URL_FILE:=dlc.dat
+  URL:=https://github.com/Loyalsoldier/v2ray-rules-dat/releases/download/$(GEOSITE_VER)/
+  URL_FILE:=geosite.dat
   FILE:=$(GEOSITE_FILE)
-  HASH:=9065ea5d6fa20acf3e1a687b1c33f8669c48bf61b4979768ce2f50fe507053ee
+  HASH:=6775adf6977712a073fe27a3bd39fc18a2ca3fcac6473b3381dbd979b3e39d9a
 endef
 
 define Package/v2ray-geodata/template
 
